### DISCUSSION TITLE
Total NOOB attempting HyperHDR install

### DISCUSSION THREAD

**Discussion by jwohyperion** [awawa-dev/HyperHDR#925] [Q&A] [answered]
I am attempting to install HyperHDR on a PI model b+ when I attempt to download HyperHDR-20.0.0.0-Linux-armv61.deb, using the command:wget https://github.com/awawa-dev/HyperHDR/releases/download/v20.0.0.0/HyperHDR-20.0.0.0-Linux-armv61.deb I am getting a "404 directory not found" error. Am I doing something wrong or is the link broken? I was able to download that file to my PC and transfer it to the PI but then I get  "unsupported file" error.
thanks

**Comment by awawa-dev** [comment, marked as answer]
Hi
'l' not '1' in ...armv6**1**.deb. Correct link:
https://github.com/awawa-dev/HyperHDR/releases/download/v20.0.0.0/HyperHDR-20.0.0.0-Linux-armv6l.deb

**Comment by jwohyperion** [comment]
Hi,
Thanks for the info, I copied and pasted into putty and got this response
-bash:
https://github.com/awawa-dev/HyperHDR/releases/download/v20.0.0.0/HyperHDR-20.0.0.0-Linux-armv6l.deb:
No such file or directory

**Reply by awawa-dev** [reply]
and where is wget?

**Comment by jwohyperion** [comment]
I'm sorry,I'm such a Noob I didn't realize I didn't get the file yet. Here
is wget command and response.


***@***.***:~ $ wget
https://github.com/awawa-dev/HyperHDR/releases/download/v20.0.0.0/HyperHDR-20.0.0.0-Linux-armv6I.deb
--2024-08-17 21:04:30--
https://github.com/awawa-dev/HyperHDR/releases/download/v20.0.0.0/HyperHDR-20.0.0.0-Linux-armv6I.deb
Resolving github.com (github.com)... 140.82.112.3
Connecting to github.com (github.com)|140.82.112.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-08-17 21:04:31 ERROR 404: Not Found.

**Reply by awawa-dev** [reply]
this is a recidivism, and a mistake made in the same place ;)

**Comment by jwohyperion** [comment]
Thanks for your patience. I still don't see what I did wrong but I appended
a "wget" to your link and download and installed successfully,
Thanks again
